project.clj

```diff
@@ -13,7 +13,8 @@
                  [secretary "1.2.3"]]
 
   :plugins [[lein-environ "1.0.2"]
-            [lein-asset-minifier "0.2.8"]]
+            [lein-asset-minifier "0.2.8"]
+            [lein-cljfmt "0.3.0"]]
```

Comment:
Current version seems to be `0.5.3` (via: https://clojars.org/lein-cljfmt). Is there a reason for using `0.3.0`?